src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/entity/LocationSchedule.kt

```diff
@@ -97,8 +97,7 @@ class LocationSchedule private constructor(
     require(startTime.isBefore(endTime)) { "The end time must come after the start time." }
 
     if (
-      locationAttribute.schedule().any {
-        it.locationScheduleId != locationScheduleId
```

Comment:
Was missing ampersand of the the check.  This should have been using a filter {} anyway, so corrected.